docs/connections/manage-etl-workflows.md

```diff
@@ -43,21 +71,48 @@ You can only add target workflows to a Macrometa document collection that contai
 8. (Optional) If you want to transform the data, then click **Add Transformation**. For more information about supported Stream QL, refer to [Transformations](./transformations).
 9. Click **Create**.
 
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to [Create a Source or Target Workflow](https://www.macrometa.com/docs/api#/operations/createWorkflow).
+
+To add a transformation:
+
+1. [Retrieve Sample Data from the Data Source](https://www.macrometa.com/docs/api#/operations/retrieveSample)
+2. [Retrieve the Schema from the Data Source](https://www.macrometa.com/docs/api#/operations/retrieveSchema)
```

Comment:
@diana-macrometa Step 1. response contain both sample data and the schema. So Step 2. Is not a mandatory step in tranformation creation. Everything else is correct in this flow

Reply:
@diana-macrometa we have to provide the transformation query created in here to `createWorkflow` request.